docs/source/json-api/index.rst

```diff
@@ -1944,14 +1944,32 @@ instead.
     {"templateIds": ["11c8f3ace75868d28136adc5cfc1de265a9ee5ad73fe8f2db97510e3631096a2:Iou:Iou"]}
 
 Multiple queries may be specified in an array, for overlapping or
-different sets of template IDs::
+different sets of template IDs.::
 
     [
         {"templateIds": ["11c8f3ace75868d28136adc5cfc1de265a9ee5ad73fe8f2db97510e3631096a2:Iou:Iou"], "query": {"amount": {"%lte": 50}}},
-        {"templateIds": ["11c8f3ace75868d28136adc5cfc1de265a9ee5ad73fe8f2db97510e3631096a2:Iou:Iou"], "query": {"amount": {"%gt": 50}}},
+        {"templateIds": ["11c8f3ace75868d28136adc5cfc1de265a9ee5ad73fe8f2db97510e3631096a2:OtherIou:OtherIou"], "query": {"amount": {"%gt": 50}}},
         {"templateIds": ["11c8f3ace75868d28136adc5cfc1de265a9ee5ad73fe8f2db97510e3631096a2:Iou:Iou"]}
     ]
 
+Interface ID can be provided in ``templateIds`` but only single interface
+ID can be used in all queries::
```

Comment:
```suggestion
An interface ID can be used in all queries::
```

Reply:
"ID can be used in all queries::" is following the previous line as 1 sentence. 
I will change it to 
```
Interface ID can be provided in ``templateIds`` but only a single interface
ID can be used in all queries::
```

Reply:
oh sorry I have mis-read. you actually made suggestion on the previous line
I will apply this